projects/packages/jetpack-mu-wpcom/src/features/100-year-plan/locked-mode.php

```diff
@@ -0,0 +1,200 @@
+<?php
+/**
+ * Locked Mode.
+ *
+ * @package automattic/jetpack-mu-wpcom
+ */
+
+/**
+ * Adds the action to limit capabilities for locked mode sites when the Locked Mode feature is available
+ * and the option enabled.
+ */
+function wpcom_lm_maybe_add_map_meta_cap_filter() {
+	if ( wpcom_site_has_feature( WPCOM_Features::LOCKED_MODE ) && get_option( 'wpcom_locked_mode' ) ) {
+		add_filter( 'map_meta_cap', 'wpcom_lm_remove_post_capabilities', 10, 2 );
+	}
+}
+add_action( 'plugins_loaded', 'wpcom_lm_maybe_add_map_meta_cap_filter' );
```

Comment:
I think `plugins_loaded` may have already fired when this code runs, so the capabilities aren't being limited. I put `error_log(did_action('plugins_loaded'));` before this line to check (it's returning `1`). But, `setup_theme` seems to work. Do you think `setup_theme` runs early enough to suffice?

Reply:
Oh, I see, this file gets loaded on `plugins_loaded`. Weird, it worked for me in wp-admin for some reason?
If you add a priority of 11 as a third parameter, does that make it work?

Reply:
Adding a priority of 11 works! That's a bit mystifying to me, because with the code below, I can see `plugins_loaded` has already fired once. So, I'd assume the priority wouldn't matter at that point. But, I guess it fires again?

```
error_log( print_r( did_action( 'plugins_loaded' ) ) );
add_action( 'plugins_loaded', 'wpcom_lm_maybe_add_map_meta_cap_filter', 11 );
```

Reply:
Fixed in https://github.com/Automattic/jetpack/pull/33081/commits/6f6829233b5bc30aa44896787dffb727ac443722